README.md

```diff
@@ -40,6 +93,18 @@ This workflow creates a release. Its trigger event should be when a new tag is c
 |--------------------|----------------------------------------------------------|---------|-----------|-----------------|
 | enableOpenApiToPostman | Flag to enable OpenAPI to Postman collection conversion                       | boolean | no        | true            |
 
+```mermaid
+flowchart TD
+    classDef head fill:#5882FA
+    A[Release On Tag Push]:::head -->|INPUTS: \n enableOpenApiToPostman| C["Checkout code for CHANGELOG.md \n (for release notes)"]
```

Comment:
`Checkout Git repository` is more precise

Reply:
Changed